### DISCUSSION TITLE
netbox using SSO ADFS

### DISCUSSION THREAD

**Discussion by zhabchenko** [netbox-community/netbox#14293] [Help Wanted!]
Hi guys, can anyone share the configuration for connecting netbox to SSO ADFS. The documentation only contains Google and Okta.

**Comment by candlerb** [comment]
To be fair, the documentation also covers [Active Directory](https://docs.netbox.dev/en/stable/installation/6-ldap/#authenticating-with-active-directory) (via LDAP) and [Azure AD](https://docs.netbox.dev/en/stable/administration/authentication/microsoft-azure-ad/).
